### ISSUE TITLE
Old migrations not found after updating

### ISSUE THREAD

**Issue by herbyxxx** [doctrine/migrations#1034]
Basically updated my symfony 5.0 to 5.1 and with that doctrine/migrations to 3.0.

After that update I had issues that my doctrine_migrations.yaml file was wrong so I changed mine to
```
doctrine_migrations:
    migrations_paths:
        'DoctrineMigrations': '%kernel.project_dir%/src/Migrations'
```

However I did notice it does not ignore my old migrations and keeps applying them to the database. Now I did find out that the default table name changed from `migration_versions` to `doctrine_migration_versions`. So after copy/paste the data from the old table into `doctrine_migration_versions` I thought it would work. However it still tries to apply old migrations and says I currently have no migration.

```
+----------------------+----------------------+------------------------------------------------------------------------+
| Configuration                                                                                                        |
+----------------------+----------------------+------------------------------------------------------------------------+
| Storage              | Type                 | Doctrine\Migrations\Metadata\Storage\TableMetadataStorageConfiguration |
|                      | Table Name           | doctrine_migration_versions                                            |
|                      | Column Name          | version                                                                |
|----------------------------------------------------------------------------------------------------------------------|
| Database             | Driver               | Doctrine\DBAL\Driver\PDOMySql\Driver                                   |
|                      | Name                 | test                                                                   |
|----------------------------------------------------------------------------------------------------------------------|
| Versions             | Previous             | 0                                                                      |
|                      | Current              | 0                                                                      |
|                      | Next                 | DoctrineMigrations\Version20190806080236                               |
|                      | Latest               | DoctrineMigrations\Version20191218125013                               |
|----------------------------------------------------------------------------------------------------------------------|
| Migrations           | Executed             | 0                                                                      |
|                      | Executed Unavailable | 0                                                                      |
|                      | Available            | 29                                                                     |
|                      | New                  | 29                                                                     |
|----------------------------------------------------------------------------------------------------------------------|
| Migration Namespaces | DoctrineMigrations   | C:\xampp\htdocs\test/src/Migrations                                    |
+----------------------+----------------------+------------------------------------------------------------------------+

```

How can I fix this issue? Thanks

**Comment by herbyxxx** [comment]
Edit: Migration names changed too. I fixed my issue by renaming `20190806080236` in my table (column version) to `DoctrineMigrations\Version20190806080236`

**Comment by digitalshow** [comment]
I hit that problem too, I use sqlite3. What I did was
```
CREATE TABLE doctrine_migration_versions (version VARCHAR(191) NOT NULL, executed_at DATETIME DEFAULT NULL, execution_time INTEGER DEFAULT NULL, PRIMARY KEY(version));
INSERT INTO doctrine_migration_versions (version, executed_at) SELECT printf("DoctrineMigrations\Version%s", version), executed_at FROM migration_versions;
```
